spec/forms/developer_search_form_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
```

Comment:
Do you set up `rubocop` and `overcommit` properly?
No `# frozen_string_literal: true`.

Reply:
I forget to run `rubocop --auto-correct`